sdk/search/Azure.Search.Documents/src/Spatial/GeoObjectProxy.cs

```diff
@@ -86,5 +87,35 @@ protected T GetPropertyValue<T>(ref PropertyInfo property, string name)
 
             return proxies;
         }
+
+        /// <summary>
+        /// Gets the proxied <see cref="Azure.Core.GeoJson.GeoPosition"/> collection value of the named property.
+        /// </summary>
+        /// <param name="property">A reference to a statically cached <see cref="PropertyInfo"/>.</param>
+        /// <param name="proxies">A reference to an cached collection of <see cref="Azure.Core.GeoJson.GeoPosition"/> proxies.</param>
+        /// <param name="name">The name of the property.</param>
+        /// <returns>A proxied <see cref="Azure.Core.GeoJson.GeoPosition"/> collection of the named property.</returns>
+        protected IReadOnlyList<GeoPositionProxy> GetCollectionPropertyValue(
+            ref PropertyInfo property,
+            ref IReadOnlyList<GeoPositionProxy> proxies,
+            string name)
+        {
+            if (proxies is null)
+            {
+                IEnumerable<GeoPosition> _list = GetPropertyValue<IEnumerable<GeoPosition>>(ref property, name);
+                var list = new List<GeoPosition>(_list);
```

Comment:
[comment copied from older PR] @heaths  : Nit: would be nice to stick with a style: var + explicit constructor, or explicit type and just new(...). These are just whitespaces apart.